docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md

```diff
@@ -0,0 +1,129 @@
+
+!!! note
+    To setup your alert receiver, with Amazon SNS, follow [this documentation](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-alertmanager-receiver.html)
+
+## Deploy an example application to visualize metrics
+
+In this section we will deploy Istio's Bookinfo sample application and extract metrics using the AWS OpenTelemetry collector. When downloading and configuring `istioctl`, there are samples included in the Istio package directory. The deployment files for Bookinfo are found in the `samples` folder. Additional details can be found on Istio's [Getting Started](https://istio.io/latest/docs/setup/getting-started/) documentation
+
+### 1. Deploy the Bookinfo Application
+
+1. Using the AWS CLI, configure kubectl so you can connect to your EKS cluster. Update for your region and EKS cluster name
+```sh
+aws eks update-kubeconfig --region <enter-your-region> --name <cluster-name>
+```
+2. Label the default namespace for automatic Istio sidecar injection
+```sh
+kubectl label namespace default istio-injection=enabled
+```
+3. Navigate to the Istio folder location. For example, if using Istio v1.18.2 in Downloads folder:
+```sh
+cd ~/Downloads/istio-1.18.2
+```
+4. Deploy the Bookinfo sample application
+```sh
+kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+```
+5. Connect the Bookinfo application with the Istio gateway
+```sh
+kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+```
+6. Validate that there are no issues with the Istio configuration
+```sh
+istioctl analyze
+```
+7. Get the DNS name of the load balancer for the Istio gateway
+```sh
+GATEWAY_URL=$(kubectl get svc istio-ingressgateway -n istio-system -o=jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+```
+
+### 2. Generate traffic for the Istio Bookinfo sample application
+
+For the Bookinfo sample application, visit `http://$GATEWAY_URL/productpage` in your web browser. To see trace data, you must send requests to your service. The number of requests depends on Istio’s sampling rate and can be configured using the Telemetry API. With the default sampling rate of 1%, you need to send at least 100 requests before the first trace is visible. To send a 100 requests to the productpage service, use the following command:
+```sh
+for i in $(seq 1 100); do curl -s -o /dev/null "http://$GATEWAY_URL/productpage"; done
+```
+
+### 3. Explore the Istio dashboards
+
+Log back into your Amazon Managed Grafana workspace and navigate to the dashboard side panel. Click on the `Observability Accelerator Dashboards` folder and open the `Istio Service` Dashboard. Use the Service dropdown menu to select the `reviews.default.svc.cluster.local` service. This gives details about metrics for the service, client workloads (workloads that are calling this service), and service workloads (workloads that are providing this service).
+
+Explore the Istio Control Plane, Mesh, and Performance dashboards as well.
```

Comment:
Also can you please add screenshots for all the 4 dashboards so people can see it.